src/Assets/component/administrator/sql/install.mysql.utf8.jext

```diff
@@ -2,6 +2,7 @@ CREATE TABLE IF NOT EXISTS `#__{{component}}_notes` (
 	`id` bigint NOT NULL AUTO_INCREMENT,
 	`asset_id` bigint NOT NULL DEFAULT 0,
 	`title` varchar(255) NOT NULL DEFAULT '',
+	`name` varchar(255) NOT NULL DEFAULT '',
```

Comment:
`name` is not a common table column. Here in the `install.mysql.sql` file, I've added the common fields which are commonly used.

Reply:
I added a.name because if you search in the bachend it gives an error because the search is done on name not title

Reply:
This is a mistake to search by the `name` field instead of `title`. So, changing the field name to the search query is okay.

And you know, I've added some common fields here and one component developer should update his/her table structure after installation. And the `name` field is not a common field in Joomla!

Reply:
thank you.